projects/angular/src/layout/_login.clarity.scss

```
@@ -300,9 +357,20 @@
           line-height: variables.$clr_baselineRem_0_75;
 
           &:before {
-            display: inline-block; //needed for IE. display: none to display:flex on parent doesnt work
             content: '';
-            background: icons.generateErrorIcon(login-variables.$clr-login-background-color);
+            display: inline-block; //needed for IE. display: none to display:flex on parent doesnt work
```

We can end this IE backward compatibility.

```suggestion
            display: flex;
```